### PR TITLE
Version bump

### PR DESCRIPTION
**Pull request by gmokki**

### CODE REVIEW

File: nflow-engine/src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
       <onMismatch>NEUTRAL</onMismatch>
     </filter>
     <encoder>
-      <pattern>%d{dd.MM.yyyy HH:mm:ss,SSS} %-5p [%t] %-30.30c - [user:%X{username}] %msg%n</pattern>
+      <pattern>%d %-5p [%t] %-30.30c - [user:%X{username}] %msg%n</pattern>
```
Comment (efonsell) [inline nflow-engine/src/test/resources/logback-test.xml:15]: Why these changes in version bump PR?

File: nflow-tests/src/test/java/io/nflow/tests/CreditApplicationWorkflowTest.java [hunk 2]
```diff
@@ -94,14 +94,14 @@ public void checkErrorStateReached() {
...
-        new Action(i++, stateExecution.name(), "acceptCreditApplication", "", 0, null, null, 0), // probably not the way to show manual action in future
-        new Action(i++, stateExecution.name(), "createCreditApplication", "", 0, null, null, 0))));
+        new Action(0, stateExecution.name(), "error", "", 0, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 3, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 2, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 1, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 0, null, null, 0),
+        new Action(0, stateExecution.name(), "grantLoan", "", 0, null, null, 0),
+        new Action(0, stateExecution.name(), "acceptCreditApplication", "", 0, null, null, 0), // probably not the way to show manual action in future
+        new Action(0, stateExecution.name(), "createCreditApplication", "", 0, null, null, 0))));
```
Comment (efonsell) [inline nflow-tests/src/test/java/io/nflow/tests/CreditApplicationWorkflowTest.java:104]: Why this change?

File: nflow-engine/src/test/resources/logback-test.xml (hunk 1 above)
Comment (gmokki) [reply nflow-engine/src/test/resources/logback-test.xml:15]: Without it the tests fail because `SSS` is not a valid timezone. This one is mentioned in the comments

File: nflow-tests/src/test/java/io/nflow/tests/CreditApplicationWorkflowTest.java (hunk 2 above)
Comment (gmokki) [reply nflow-tests/src/test/java/io/nflow/tests/CreditApplicationWorkflowTest.java:104]: it was unused. So let's initialize it to null/zero value like the other fields in the actions.
Noticed it because I had one random failure in the tests and had  spared a glance at the code.
I can revert this change from the PR since it is not needed for the version bump

Comment (efonsell) [reply nflow-tests/src/test/java/io/nflow/tests/CreditApplicationWorkflowTest.java:104]: Ok to keep this then

File: nflow-engine/src/test/resources/logback-test.xml (hunk 1 above)
Comment (efonsell) [reply nflow-engine/src/test/resources/logback-test.xml:15]: ok

**Review by efonsell — APPROVED** [review]
